### PR TITLE
Fix bpf logs

### PR DESCRIPTION
**Pull request by fasaxc**
## Description

* Fix spammy (and expected) "Dump was interrupted" error with retry (which should then see the link as gone).
* Fix spammy (and expected) "Link not found" errors
* Add missing log summarisation for the BPF components.

## Todos
- [ ] Unit tests (full coverage)
- [ ] Integration tests (delete as appropriate) In plan/Not needed/Done
- [ ] Documentation
- [ ] Backport
- [ ] Release note

## Release Note

```release-note
Reduce log spam from eBPF dataplane and add eBPF dataplane components to loop summary.
```

### CODE REVIEW

**Review by neiljerram — APPROVED** [review]
Looks good.

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
